src/runtimes/node/detect_es_module.js

```diff
@@ -0,0 +1,22 @@
+const fs = require('fs')
+
+const { init, parse } = require('es-module-lexer')
+
+const detectEsModule = async ({ srcFile }) => {
+  // console.log('srcFile: ', srcFile)
```

Comment:
not quite sure yet how to accurately supply this param until I figure out exactly what the params to `zipFunction` each mean